src/rgw/rgw_lc.cc

```diff
@@ -147,10 +147,9 @@ void *RGWLC::LCWorker::entry() {
 
     utime_t end = ceph_clock_now();
     int secs = schedule_next_start_time(start, end);
-    time_t next_time = end + secs;
-    char buf[30];
-    char *nt = ctime_r(&next_time, buf);
-    dout(5) << "schedule life cycle next start time: " << nt <<dendl;
+    end += secs;
```

Comment:
Actually I think it was me being lazy, and not overloading operator + with float for `utime_t` class, I'll take a look, might be a worthy addition